app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -217,3 +220,9 @@ document.addEventListener "turbolinks:load", ->
     history.pushState({}, '', this.href);
     $(this.hash)[0].scrollIntoView behavior: 'smooth'
     return
+
+# Un-bind fileUpload on page unload.
+document.addEventListener 'turbolinks:before-cache', ->
```

Comment:
I think this code needs to be moved closer to the initialiser block:

- It saves the `if ().length` check
- It makes it easier to understand
- It allows us to potentially extract the entire behaviour elsewhere.